hc/api/models.py

```diff
@@ -114,7 +114,8 @@ def to_dict(self):
             "tags": self.tags,
             "timeout": int(self.timeout.total_seconds()),
             "grace": int(self.grace.total_seconds()),
-            "n_pings": self.n_pings
+            "n_pings": self.n_pings,
+            "status": self.status
```

Comment:
Your are definitely right about `get_status()`. I will change that. 

But I'am not sure about returning `grace` as status, since it is not really a status of a check according to https://github.com/healthchecks/healthchecks/blob/master/hc/api/models.py#L16-L21.